README.md

```diff
@@ -15,12 +15,12 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js" id="_fed_an_ua_tag"></script>
+<script language="javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=AGENCY" id="_fed_an_ua_tag"></script>
```

Comment:
`language="javascript"` is optional in all browsers, I'm pretty sure -- but I can see why you'd want to keep it consistent with the implementation guide. So consider that a note for the implementation guide. =)